### ISSUE TITLE
Won't start because of Harmony runtime parameters

### ISSUE THREAD

**Issue by svendahlstrand** [meeDamian/toggl-cli#7]
I can't run `toggl` because of two harmony runtime parameters. If I remove them from `index.js` everything works fine.

According to [node.green](http://node.green) destructuring and rest parameters should not need harmony flags to run on node 6.2.

I've tried this on OS X 10.11.5 and Raspbian Jessie Lite May 2016:

```sh
$ toggl
/usr/local/bin/node: bad option: --harmony_destructuring
/usr/local/bin/node: bad option: --harmony_rest_parameters

$ node -v
v6.2.0
```

**Comment by meeDamian** [comment]
Hey, thanks for info! I'm aware of that, but didn't have the time to fix it properly yet. 

For now, you can either `nvm use 5` or remove flags from `index.js` :)

**Comment by meeDamian** [comment]
Fixed in ffeb1902b04adf1c12068c1f04f88dcbb2eb163f

**Comment by svendahlstrand** [comment]
Thanks for fixing this so fast, @meeDamian! 😄 Now starts just fine but I get this output when I run `toggl`:

```sh
$ toggl
/usr/local/bin/toggl: 2: /usr/local/bin/toggl: [[: not found
```

**Comment by meeDamian** [comment]
is this on osx or raspbian?

**Comment by svendahlstrand** [comment]
It's on Raspbian. I see now that `index.js` is using the Bourne Shell and running [ShellCheck][1] on it gives the following result:

<img width="990" alt="sc" src="https://cloud.githubusercontent.com/assets/35639/15528303/dcd94a8a-2242-11e6-8f32-a4dc85772517.png">

So it seems like the double bracket thing is [undefined behavior in `sh`][2]. My guess is that's what causing the problem.

The OS X installation I'm on seems to use `bash` even for `sh`:

```sh
$ sh --version
GNU bash, version 3.2.57(1)-release (x86_64-apple-darwin15)
Copyright (C) 2007 Free Software Foundation, Inc.
```

[1]: https://www.shellcheck.net
[2]: https://github.com/koalaman/shellcheck/wiki/SC2039

**Comment by meeDamian** [comment]
Thanks for `shellcheck` tool!

Should be fixed in ac634d328e6a97ce00d5a2280b4a92c19491a6aa.
